docs/usage/controls.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Controls
+order: 4
+---
+
+# Form controls
+
+Controls as we use the term here refer to the UI widgets that allow a user to enter data. In its most basic form that would be an `<input>`.
+
+ember-headless-form comes with support for the following controls built-in (but you can also use [custom controls](./custom-controls/index.md)), all yielded from the `Field` component:
+
+## Input
+
+Renders a basic `<input>` element. Set `@type` to any of the supported [input types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types) other than the default `text`.
+
+Note that `checkbox` and `radio` should not be used, as they have dedicated control components (see below).
+Also these types are not useful to use as input controls: `button`,`file`,`image`,`reset`,`submit`.
+
+```hbs preview-template
+<HeadlessForm as |form|>
+  <form.Field @name='email' as |field|>
+    <div class='my-2 flex flex-col'>
+      <field.Label>Email</field.Label>
+      <field.Input
+        @type='email'
+        placeholder='Enter your email address'
+        class='border rounded px-2'
+      />
+    </div>
+  </form.Field>
+
+  <form.Field @name='birthday' as |field|>
+    <div class='my-2 flex flex-col'>
+      <field.Label>Birthday</field.Label>
+      <field.Input @type='date' class='border rounded px-2' />
```

Comment:
Another dark mode styling issue on all of these components (textarea, etc.)- looks great in light mode though!

<img width="1007" alt="Screenshot 2023-02-24 at 2 05 17 PM" src="https://user-images.githubusercontent.com/8069555/221269194-ebf4dfac-230a-4241-aebd-bea9d0add581.png">